### DISCUSSION TITLE
Mencione que tipo de software quisiera llegar a crear algun dia

### DISCUSSION THREAD

**Discussion by efonsecab** [efonsecab/Mentoring-Blazor-y-mas#2] [General]
Piense en algun tipo de software que usted quisiera llegar a crear alguna vez, y expliquelo aca.
Mencione ademas si ha hecho algun intento por crearlo, y que le ha impedido tenerlo ya terminado.

**Comment by kensolano** [comment]
Una vez me pidieron un prototipo de software para una empresa de mercadeo multinivel y no pude crearlo, siempre he querido lograr crear algo así, también me gustaría hacer una aplicación web para una empresa de mensajería (entrega de paquetes). La limitación en ambos casos es que no se aplicar C# a web y tampoco se frontend (tengo un nivel muy básico de frontend y no domino muy bien el CSS), también, en un nivel más avanzado me gustaría hacer alguna plataforma de streaming y páginas de juegos.  
También tengo un proyecto personal de crear videojuegos.

**Comment by Gust23ccCastillo** [comment]
Me gustaría crea una aplicación como lo es Didi foots pero en la cual no solo se venda comidas si no de todo un poco ferreteria,ropa , comida considero que seria una buena aplicación además se abarcaría diferentes empresas locales del pais que se beneficiarían al tenerse virtualmente y que el usuario sin salir de casa consiga lo que busca, pero considero que todavía mis conocimientos no esta de todo bien fomentados como para crear una aplicación  de ese nivel me falta ciertas cosas que aprehender pero como siempre lo eh dicho "SI ALGUIEN PUEDE POR QUE YO NO".

**Comment by EithanMM** [comment]
Algún tipo de aplicación que solvente una necesidad actual en el área de salud, ya sea que agilice un proceso que actualmente es manual, pero que suceda en varias áreas de salud. O generar una API lo suficientemente útil como para que varios sistemas o appliaciones necesiten consultar dicha data constantemente y monetizar cada llamado a la API.

**Comment by JoseIgnacioChavarriaBrenes** [comment]
Me encantaría  crear un videojuego los mas realista posibles en el apartado de la inteligencia artificial y de las físicas

**Comment by Yahairacc** [comment]
Me gustaria hacer una aplicacion para ayudar a las personas sordas a comunicarse soy madre de una niña sorda y a pesar de que se han creado o han querido crear apps para sordos ninguna es suficientemente buena para dar una verdadera ayuda a esta poblacion

**Comment by mardeanto** [comment]
pensaría en uno que me ayude en el bienestar diario, como mejorar el uso del tiempo. También otro que pueda mejorar mis finanzas o dar opciones para incrementarlas.
